ghci/containers/ghci-koji/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.fedoraproject.org/fedora:32
+
+# koji db schema is in docs, remove nodocs from from dnf config
+RUN sed  -i '/^tsflags=nodocs$/d' /etc/dnf/dnf.conf \
+    && dnf \
+            -y \
+            --setopt=fastestmirror=True \
+            --setopt=install_weak_deps=False \
+            install koji-hub postgresql \
+    && dnf clean all
```

Comment:
```suggestion
# koji db schema is in docs, remove nodocs from from dnf config
RUN sed  -i '/^tsflags=nodocs$/d' /etc/dnf/dnf.conf

RUN dnf -y upgrade \
    && dnf -y \
            --setopt=fastestmirror=True \
            --setopt=install_weak_deps=False \
            install \
                    koji-hub \
                    postgresql \
    && dnf clean all
```

You sure you don't want an `dnf -y upgrade` in there? If the previous stage is cached, you might have an out-of-date system and thus the `install` might fail because your local dependencies are wrong. This is unlikely to happen, but also easy to avoid. It does increase build-times a bit, but... meh...

Again, fine with me if you ignore it. I was just wondering...

Reply:
Yeah, I thought that the images from the fedora registry are updated, my bad, I added the `dnf upgrade`.